src/lengths.jl

```diff
@@ -1,19 +1,20 @@
-export gyroradius
+export gyroradius, rc_
+
+const rc_ = gyroradius
 
 """
 Calculate the radius of circular motion for a charged particle in a uniform magnetic field
+
+References: https://docs.plasmapy.org/en/latest/api/plasmapy.formulary.lengths.gyroradius.html
 """
-function gyroradius(B, mass_numb, Z, Vperp::Unitful.Velocity)
+function gyroradius(B, Vperp::Velocity; mass_numb=1, Z=1)
```

Comment:
I like `mass_numb` better as a keyword arg, although consider using the full words: `mass_number`.

Reply:
I also love `mass_number` but `plasmapy` uses `mass_numb` which makes my life hard

Reply:
> I also love `mass_number` but `plasmapy` uses `mass_numb` which makes my life hard

For PlasmaPy, we ended up using `mass_numb` as the parameter name for functions because we have a function in `plasmapy.particles` called `mass_number`, and we wanted to avoid a namespace overlap. I was a little ambivalent about this design decision since I usually prefer to avoid abbreviations in variable names.

I'm excited to see this package! 🎉🎂🎆🪩🌌🥦

Reply:
Hi @namurphy Thank you for sharing your thoughts! I also prefer to avoid abbreviations and this function actually got refactored later with`particlelike` parameter.

Since you are here, I am just wondering why in plasampy we use `Z` instead of `z`, as I think it is more common to use `Z` for atomic number

Reply:
I believe we chose capital `Z` for the charge number because the NRL formulary uses `Z` instead of `z`. It might also have been because `Z` is used more often in fields like heliophysics (where several of the original contributors came from) and a few other subfields of plasma physics. The differences in terminology and variable conventions across different parts of the plasma community have indeed led to some headaches and confusion over the years...

Reply:
Thanks for clarifying the terminology! It really helps me understand the deisgn better.